### ISSUE TITLE
Unable to initialize Microsoft Edge driver on macOS with default capabilities

### ISSUE THREAD

**Issue by maxim-lobanov** [SeleniumHQ/selenium#7979]
## 🐛 Bug Report

Microsoft Edge browser and Microsoft Edge WebDriver are available on macOS.
However, if you write any simple selenium test, driver initialization will fail on macOS because WINDOWS platform is hardcoded in capabilities.
Issue affects Python and JavaScript.

## To Reproduce

1) Use macOS 10.15 (or any other macOS)
2) Initialize Edge driver with default settings on Python
```
def setUp(self):
        self.driver = webdriver.Edge()
```
The following errors will appear:
```
selenium.common.exceptions.SessionNotCreatedException: Message: session not created: No matching capabilities found
```

## Expected behavior

I expect that `msedgedriver` will be initialized successfully with default settings

## Workaround
We can override default EDGE capability
```
def setUp(self):
        cap = DesiredCapabilities().EDGE
        cap["platform"] = "ANY"
        self.driver = webdriver.Edge(executable_path="/usr/local/bin/msedgedriver", capabilities=cap)
```

## Environment

OS: macOS
Browser: Edge
Browser version:  79.0.309.71 (actually, reproduces on any version)
Browser Driver version:  79.0.309.71

**Comment by maxim-lobanov** [comment]
Pull-request with fix: https://github.com/SeleniumHQ/selenium/pull/7980

**Comment by diemol** [comment]
Closed via #7980, thanks @maxim-lobanov
